app/javascript/hyacinth_v1/components/batch_imports/BatchImportIndex.js

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+import { Card, Button } from 'react-bootstrap';
+import * as moment from 'moment';
+import { useQuery, useMutation } from '@apollo/react-hooks';
+import { startCase } from 'lodash';
+
+import { Can } from '@hyacinth_v1/utils/abilityContext';
+import ContextualNavbar from '../shared/ContextualNavbar';
+import GraphQLErrors from '../shared/GraphQLErrors';
+import PaginationBar from '../shared/PaginationBar';
+import { batchImportsQuery, updateBatchImportMutation, deleteBatchImportMutation } from '../../graphql/batchImports';
+
+function BatchImportIndex() {
+  const limit = 10;
+  const [offset, setOffset] = useState(0);
+
+  const {
+    loading, error, data, refetch,
+  } = useQuery(batchImportsQuery, { variables: { limit, offset } });
+
+  const [updateBatchImport, { error: updateError }] = useMutation(updateBatchImportMutation);
+  const [deleteBatchImport, { error: deleteError }] = useMutation(deleteBatchImportMutation);
+
+  if (loading) return (<></>);
+  if (error) return (<GraphQLErrors errors={error} />);
+
+  const onPageNumberClick = (newOffset) => {
+    setOffset(newOffset);
+    refetch();
+  };
+
+  const onCancel = (id) => {
+    const variables = { input: { id, cancelled: true } };
+    updateBatchImport({ variables }).then(refetch);
+  };
+
+  const onDelete = (id) => {
+    const variables = { input: { id } };
+    deleteBatchImport({ variables }).then(refetch);
+  };
+
+  const { batchImports: { nodes: batchImports, totalCount: totalBatchImports } } = data;
+
+  return (
+    <>
+      <ContextualNavbar
+        title="Batch Imports"
+        rightHandLinks={[
+          { link: 'batch_imports/new', label: 'New Batch Import' },
+        ]}
+      />
+      <GraphQLErrors errors={updateError || deleteError} />
+      { batchImports.length === 0 && (<p className="text-center">No Batch Imports have been created.</p>) }
+      {
+        batchImports.map(batchImport => (
+          <Card key={batchImport.id} className="mb-3">
+            <Card.Header>
+              {`Import ID: ${batchImport.id}`}
+              <div className="float-right">
+                {moment(batchImport.createdAt).format('MMMM Do YYYY, h:mm:ss a')}
```

Comment:
Isn't `moment` great?  I really like this module.

Reply:
I just saw got up to your ReadableDate component in the PR, so I'll add this suggestion:
```suggestion
                <ReadableDate date={batchImport.createdAt} />
```
(plus the import at the top of the file)

Reply:
Whoops, I'll update this to use the component.